stumpy/core.py

```diff
@@ -568,6 +569,9 @@ def check_window_size(m, max_size=None):
     max_size : int, default None
         The maximum window size allowed
 
+    n : int, default None
+        The length of the time series in a self-join case
```

Comment:
"The length of the time series in the case of a self-join. `n` should be set to `None` in the case of an AB-join.